CMakeLists.txt

```diff
@@ -452,6 +452,10 @@ if(cmake_generator_tolower MATCHES "makefile")
     message(STATUS "  PATH        | ${IV_DIR}")
     message(STATUS "  INC         | ${IV_INCLUDE_DIR}")
     message(STATUS "  X11 (INC)   | ${X11_INCLUDE_DIR}")
+    if(IV_ENABLE_X11_DYNAMIC)
```

Comment:
Is this new option being added in Interviews? Asking because I don't see any PR in iv repo.

Reply:
The https://github.com/neuronsimulator/iv/pull/22 dynam branch was merged earlier. This pull request makes use of that InterViews extension.  The exrternal/iv submodule needs updating. It also needs an explanation of how to provide a cmake configure for NEURON that tickles the cmake configure for the interviews submodule into using a cmake with -DIV_ENABLE_X11_DYNAMIC=ON and builds Interviews first so that the NEURON cmake knows that ivx11_dyload is available and libivx11dynam.so is available to copy into the same
location as libnrniv.so (if iv was built statically)

Reply:
Ah ok...I missed that PR.